services/drone/drone-build.tester.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const Joi = require('joi')
+const { isBuildStatus } = require('../build-status')
+const { ServiceTester } = require('../tester')
+
+const t = (module.exports = new ServiceTester({
```

Comment:
Should be able to replace this with the `createServiceTester` helper  function, see https://github.com/badges/shields/blob/master/doc/service-tests.md#1-boilerplate for more info

Reply:
Done! Thanks, I briefly looked at Wercker but didn't notice this.